loadtest/publish.py

```diff
@@ -43,6 +43,11 @@ def setup_api_endpoint():
                 'either uncommenting the right line in the file or specifying '
                 'the env variable API_ENDPOINT.'
                 )
+
+    if _API[-1] == '/':
+        # Remove trailing slash if present.
+        _API = _API[:-1]
+
```

Comment:
I know it would be better to have this in a library but I don't have time to do it now :(